### ISSUE TITLE
aws_eks_cluster not exporting arn attribute as per docs

### ISSUE THREAD

**Issue by razaj92** [terraform-providers/terraform-provider-aws#4764]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
`Terraform v0.11.7`
`+ provider.aws v1.22.0`

### Affected Resource(s)

* aws_eks_cluster

### Terraform Configuration Files

```hcl
resource "aws_eks_cluster" "eks" {
  name     = "${var.name}"
  role_arn = "${var.role_arn}"

  vpc_config {
    subnet_ids = ["${var.subnet_ids}"]
  }
}

output "arn" {
  value = "${aws_eks_cluster.eks.arn}"
}

```

### Expected Behavior

Plan successfully, and output arn on apply as per https://www.terraform.io/docs/providers/aws/r/eks_cluster.html#arn

### Actual Behavior

`* output.arn: Resource 'aws_eks_cluster.eks' does not have attribute 'arn' for variable 'aws_eks_cluster.eks.arn'`

### Steps to Reproduce

1. `terraform plan`

### Important Factoids

https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_eks_cluster.go#L191

Missing fetch of arn in function and no value in resource schema

**Comment by razaj92** [comment]
sorry 🤦‍♂️ Didnt realise we only export `arn` from a data resource

**Comment by bflad** [comment]
Yikes! You are correct! Sorry you ran into this. This was a last minute addition from preview to GA of EKS. I'll submit the PR fix for this immediately.

**Comment by bflad** [comment]
Bug fix PR: #4766

**Comment by razaj92** [comment]
Ahh sorry hadn't seen you already raised a PR 🙈 @bflad

**Comment by bflad** [comment]
No worries @razaj92! Sorry I missed that during the release yesterday. 😅

**Comment by bflad** [comment]
This has been released in [version 1.23.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1230-june-14-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
